chromadb/test/db/test_system.py

```diff
@@ -27,20 +27,26 @@
         topic="persistent://test-tenant/test-topic/00000000-0000-0000-0000-000000000001",
         metadata={"test_str": "str1", "test_int": 1, "test_float": 1.3},
         dimension=128,
+        database=DEFAULT_DATABASE,
+        tenant=DEFAULT_TENANT,
     ),
     Collection(
         id=uuid.UUID(int=2),
         name="test_collection_2",
         topic="persistent://test-tenant/test-topic/00000000-0000-0000-0000-000000000002",
         metadata={"test_str": "str2", "test_int": 2, "test_float": 2.3},
         dimension=None,
+        database=DEFAULT_DATABASE,
+        tenant=DEFAULT_TENANT,
     ),
     Collection(
         id=uuid.UUID(int=3),
         name="test_collection_3",
         topic="persistent://test-tenant/test-topic/00000000-0000-0000-0000-000000000003",
         metadata={"test_str": "str3", "test_int": 3, "test_float": 3.3},
         dimension=None,
+        database=DEFAULT_DATABASE,
```

Comment:
should we overload the constructor with defaults instead? Also if anyone is relying on Collection() this will be backwards compat.

Reply:
The issue is that Collection here is `TypedDict` I did try, but mypy yells at me that I can't default on TypedDict fields. So yes, this feels hacky, but I can create a fixture for generating those. The unfortunate side effect is that adding this to the Collection makes things not backwards-compatible. We could convert this to a tuple or something that can have default field values and is backwards-compatible. e.g.

```python
class Collection(NamedTuple):
    id: UUID
    name: str
    topic: str
    metadata: Optional[Metadata] = None
    dimension: Optional[int] = None
    tenant: Optional[str] = None
    database: Optional[str] = None
```
wdyt?

Reply:
I can confirm that NameTuple while nice won't work due to `collection["name"]` type of access across the codebase.